aws-ts-stepfunctions/index.ts

```diff
@@ -6,63 +6,63 @@ import * as pulumi from "@pulumi/pulumi";
 const region = aws.config.requireRegion();
 
 const lambdaRole = new aws.iam.Role("lambdaRole", {
-    assumeRolePolicy: aws.iam.assumeRolePolicyForPrincipal({ Service: "lambda.amazonaws.com" }),
+  assumeRolePolicy: aws.iam.assumeRolePolicyForPrincipal({ Service: "lambda.amazonaws.com" }),
 });
 
 const lambdaRolePolicy = new aws.iam.RolePolicy("lambdaRolePolicy", {
-    role: lambdaRole.id,
-    policy: {
-        Version: "2012-10-17",
-        Statement: [{
-            Effect: "Allow",
-            Action: [
-                "logs:CreateLogGroup",
-                "logs:CreateLogStream",
-                "logs:PutLogEvents",
-            ],
-            Resource: "arn:aws:logs:*:*:*",
-        }],
-    },
+  role: lambdaRole.id,
+  policy: {
+    Version: "2012-10-17",
+    Statement: [{
+      Effect: "Allow",
+      Action: [
+        "logs:CreateLogGroup",
+        "logs:CreateLogStream",
+        "logs:PutLogEvents",
+      ],
+      Resource: "arn:aws:logs:*:*:*",
+    }],
+  },
 });
 
 const sfnRole = new aws.iam.Role("sfnRole", {
-    assumeRolePolicy: aws.iam.assumeRolePolicyForPrincipal({ Service: `states.${region}.amazonaws.com` }),
+  assumeRolePolicy: aws.iam.assumeRolePolicyForPrincipal({ Service: `states.${region}.amazonaws.com` }),
 });
 
 const sfnRolePolicy = new aws.iam.RolePolicy("sfnRolePolicy", {
-    role: sfnRole.id,
-    policy: {
-        Version: "2012-10-17",
-        Statement: [{
-            Effect: "Allow",
-            Action: [
-                "lambda:InvokeFunction",
-            ],
-            Resource: "*",
-        }],
-    },
+  role: sfnRole.id,
+  policy: {
+    Version: "2012-10-17",
+    Statement: [{
+      Effect: "Allow",
+      Action: [
+        "lambda:InvokeFunction",
+      ],
+      Resource: "*",
+    }],
+  },
 });
 
-const helloFunction = new aws.serverless.Function(
-    "helloFunction",
-    { role: lambdaRole },
-    (event, context, callback) => {
-        callback(null, "Hello");
-    },
-);
+const helloFunction = new aws.lambda.CallbackFunction(
```

Comment:
You can just use `aws.lambda.Function` instead here. I think the `CallbackFunction` is for this situation https://www.pulumi.com/docs/guides/crosswalk/aws/lambda/#specifying-attributes-on-your-magic-lambda-function

Reply:
Does ```aws.lambda.Function``` support in-line and/or magic functions in node? My understanding was it supported a type of deployment package, whether that be a local app, s3 bucket, etc.

Reply:
Got it. Yes you're right